src/debug_menu.h

```diff
@@ -26,6 +69,9 @@ void wishmutate( player *p );
 void wishskill( player *p );
 void mutation_wish();
 void draw_benchmark( const int max_difference );
+static int debug_menu_uilist();
```

Comment:
```suggestion
```

Reply:
I mean it: you don't need this line at all.

Reply:
apologies: i meant to take it out all in one pass, but i missed it. now it's out for real.